modules/motus/merge/main.nf

```diff
@@ -0,0 +1,39 @@
+VERSION = '3.0.1'
```

Comment:
Indeed I think this is an issue for current module design. This version number should be checked if user is using container or conda. Otherwise, will it be better to ask user input the version number?

Reply:
What is the difference between container or conda here? The version in the container should be the same as the bioconda version

i.e.

```
    conda (params.enable_conda ? "bioconda::motus=3.0.1" : null)
    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
        'https://depot.galaxyproject.org/singularity/motus:3.0.1--pyhdfd78af_0':
        'quay.io/biocontainers/motus:3.0.1--pyhdfd78af_0' }"
```

Also I saw in an older versio nof mOTUs I had installed, runnign just `motus --help` (or `motus`) would display the version at the top. Is that not the same now in 3.0.1 (conda doesn't want to finish the transaction when I install the new one :facepalm: )

Reply:
Or even:

```
$ motus --version
motus 3.0.1 on python 3.9.9
```

Why did you not use that originally?